addon-test-support/audit.js

```diff
@@ -5,6 +5,9 @@ import RSVP from 'rsvp';
 import config from 'ember-get-config';
 import formatViolation from 'ember-a11y-testing/utils/format-violation';
 import violationsHelper from 'ember-a11y-testing/utils/violations-helper';
+import utils from './utils';
+
+const { performance, measure } = utils;
```

Comment:
Maybe we could import these as named functions (instead of destructuring), what do you think?

```js
import { mark, measure } from './performance-utils';
```

We could even make those methods resilient to when `performance` is not defined, so we could remove the conditionals in this file...

Reply:
Yes, that's much cleaner now.